app/components/works/embargo_component.html.erb

```diff
@@ -39,7 +39,7 @@
   <div class="mb-3 row">
     <%= form.label :access, 'Who can access?', class: 'col-sm-2 col-form-label' %>
     <div class="col-sm-10">
-      <%= form.select :access, [['Stanford Community', 'stanford'], ['Everyone', 'world']], { selected: "", disabled: "", prompt: "Select..." }, class: "form-select", required: true %>
+      <%= form.select :access, [['Stanford Community', 'stanford'], ['Everyone', 'world']], { disabled: "", prompt: "Select..." }, class: "form-select", required: true %>
```

Comment:
Can you say more about this change?

Reply:
This was forcing it to reset to the prompt option.  We want it to use the value the user selected originally.

Reply:
:beers: